### ISSUE TITLE
Getting the original SQL query from `PreparedQuery`

### ISSUE THREAD

**Issue by jgonera** [adelsz/pgtyped#292]
I have a wrapper function that I use to run my queries. For now all it does is the camelCase <-> snake_case conversion but I'd like to also add some logging to it printing out which query was run and how long it took. I'm passing only a `PreparedQuery` to my function and there's no way to get the original SQL from that.

From what I can see, this could be easily achieved by switching `private readonly query` to just `readonly query` and then using `query.statement.body` to get the SQL. Would that be acceptable or should this information be stored/accessed differently?

**Comment by adelsz** [comment]
Yes, changing the `query` field visibility is a perfectly valid solution for this.
